src/data.ts

```diff
@@ -1,31 +1,131 @@
-    answer:
-      "The schedule will be available on the website soon along with our list of speakers and sponsors! We are working around the clock to create the best conference experience for you, promised. ",
+    answer: [
+      "The schedule will be available on the website soon along with our list of speakers and sponsors!",
+      "We are working around the clock to create the best conference experience for you, promised. ",
+    ],
   },
   {
     question: "How can I get a ticket?",
-    answer: "The online conference will be free of charge this year!",
+    answer: [
+      "The online conference will be free of charge this year! ",
+      "You will have access to all talks from speakers, networking in a virtual career fair with sponsors, social events, workshops hosted by sponsors, and more!",
+    ],
+  },
+  {
+    question: "What is the VIP ticket?",
+    answer: [
+      "With a VIP ticket, you will have all the perks of the free ticket as well as CUSEC 2021 merchandise, exclusive and priority networking, access to a private communication platform with the sponsors and speakers, Q&A events at the end of each night with speakers and sponsors!",
+    ],
+  },
+  {
+    question: "Why do I need to register if it's a free conference?",
+    answer: [
+      "By registering, we will collect your resumes and forward them to our sponsors and you will receive the link to sign up for the virtual conference on Hoping.",
+    ],
+  },
+  {
+    question: "How will CUSEC be delivered this year?",
+    answer: [
+      "CUSEC 2021 will be delivered through an online platform known as Hopin. Hopin was founded in 2019 by Johnny Boufarhat in London, United Kingdom. The platform began as a simple networking tool for matching people one-on-one but rapidly grew into a full-conference online events platform with Sessions, Stages, Expos, and more.",
+      "Today, Hopin is the first all-in-one live online events platform where attendees can learn, interact, and connect with people from anywhere in the world. Whether it’s a meetup of 100 people or a full-scale multi-track conference of 100,000 participants, Hopin’s a live video platform makes online events more personal and engaging.",
+      "Hopin is designed to replicate an in-person event experience, only without the barriers of travel, venues, weather, awkward wandering, parking, and so on. With Hopin, businesses, communities, and organizations can reach their global audiences, gather in one place, and make an enormous online event feel small again.",
+    ],
+  },
+  {
+    question: "How is Hopin different from a webinar?",
+    answer: [
+      "At a webinar, only a few people talk. Everybody else listens. Passively. When one person talks for an hour, and 50 or so people are on mute, it feels a little one-sided, not like an interactive event.",
+      "Bystander effect takes place -- everybody starts to think the other people on the call will volunteer to speak up.",
```

Comment:
Consider using a long dash ("—") instead of "--" like the end of this answer to be more consistent